### ISSUE TITLE
Tectonic Console x509 error

### ISSUE THREAD

**Issue by mailiam** [coreos/tectonic-forum#38]
### Tectonic Version 

1.5.2-tectonic.1

### Environment

PXE booted VM behind pfSense+HAProxy on ESXi

### Expected Behavior

Tectonic Console should brought up and Tectonic Installer completed.

### Actual Behavior

```bash
$ kubectl logs tectonic-console-1953687031-291k -n tectonic-system
```
returns 
```
2017/01/30 19:48:33 http: Provider config sync still failing, retrying in 32s: Get https://tectonic.{domain}/identity/.well-known/openid-configuration: x509: certificate signed by unknown authority
```

### Reproduction Steps

  1. Run Tectonic Installer on Ubuntu over same hypervisor.
  2. Select 'Generate a CA certificate and key for me. (default)' on http://127.0.0.1/define/certificate-authority
  3. Proceed through http://127.0.0.1/boot/connect

### Other Information

* bootcfg tls has been created with 1.5.1-tectonic.1
* `$ journcalctl -u bootkube` completed with 'Done'
* all other pods are running
*  `$ openssl s_client -CAfile ./assets/tls/ca.crt -connect tectonic.example.com:443 | openssl x509 -text` returns 
```
depth=0 O = kube-aws, CN = tectonic.jasang.net
verify error:num=20:unable to get local issuer certificate
verify return:1
depth=0 O = kube-aws, CN = tectonic.jasang.net
verify error:num=27:certificate not trusted
verify return:1
depth=0 O = kube-aws, CN = tectonic.jasang.net
verify error:num=21:unable to verify the first certificate
verify return:1
Certificate:
    Data:
...
...
```

**Comment by ericchiang** [comment]
Does `https://tectonic.{domain}/` go through your proxy? How does it handle TLS traffic?

**Comment by dghubble** [comment]
Does your tectonic domain name (`tectonic.jasang.net`) resolve to any of your worker node(s)? If you use the `kubeconfig` from the assets bundle, can you see if all pods are running?

```
kubectl --kubeconfig=assets/auth/kubeconfig get pods --all-namespaces
```

https://coreos.com/tectonic/docs/latest/admin/assets-zip.html

**Comment by mailiam** [comment]
@ericchiang
Thank you for the response,
https://tectonic.{domain}/identity/.well-known/openid-configuration does resolve correctly with above tls 

How would I test TLS traffic?
Accessing through Firefox complains that the certificate is not secure.

**Comment by mailiam** [comment]
@dghubble 

Other pods' status via `kubectl --kubeconfig=assets/auth/kubeconfig get pods --all-namespaces` looks fine.

I have tried below commands from provided [reference](https://coreos.com/tectonic/docs/latest/admin/assets-zip.html) but have no luck resolving the issue.
```
unzip assets.zip
export KUBECONFIG=$PWD/assets/auth/kubeconfig
kubectl apply -f ./assets/tectonic/identity-config.yaml
```
```
kubectl patch deployment tectonic-identity \
    --patch "{\"spec\":{\"template\":{\"metadata\":{\"annotations\":{\"date\":\"`date +'%s'`\"}}}}}" \
    --namespace tectonic-system
```

**Comment by mfburnett** [comment]
cc @ivancherapau

**Comment by IvanCherepov** [comment]
@mailiam 
- double check that `tectonic.jasang.net` resolve to any of your worker nodes. 
- you reported `1.5.2-tectonic.1` for tectonic version but `1.5.1-tectonic.1` for bootcfg. I'm not sure what version you're using but it's better to use the latest version.

**Comment by pompomJuice** [comment]
This is the same issue, those certs are buggered.

**Comment by mfburnett** [comment]
@mailiam were you able to get this issue resolved with the latest release of Tectonic 1.6.2-tectonic.1?

**Comment by mailiam** [comment]
@mfburnett 
I've moved on to Rancher.
I will try again when I have a chance to fire up new k8s cluster.

You may close the issue for now.
